packages/console-extension/schema/tracker.json

```diff
@@ -37,6 +37,13 @@
       "type": "string",
       "enum": ["notebook", "terminal"],
       "default": "notebook"
+    },
+    "kernelActivity": {
+      "title": "Kernel activity",
+      "description": "Whether the console should show all or minimal kernel activity",
```

Comment:
It isn't clear what "minimal" means, especially since that option does not occur anywhere else in the UX. What do you think of making this a boolean and changing the description to just "Whether the console should show all kernel activity by default", with a default of false? That would match the option in the ux to "Show all kernel activity".

Being a boolean does limit future flexibility, but it also does match what the ux currently provides.

Reply:
Similarly, can we make the setting name and title more descriptive, like `showAllKernelActivity` or something?